### ISSUE TITLE
set_time

### ISSUE THREAD

**Issue by wimmatthijs** [janjongboom/saml21-lorawan-mbedos5#3]
set_time doesn't seem to work,
keep getting 01/01/1970 no matter what has been set :-)
does the lib need to be adapted or am i doing something wrong? ;)

**Comment by janjongboom** [comment]
@wimmatthijs Yes, the RTC is not working for this target... See the test report in https://github.com/ARMmbed/mbed-os/pull/8658.

If you have any idea on how to get the RTC running (I assume some clock config is not set correctly) I'd be happy to take it in.

**Comment by wimmatthijs** [comment]
@janjongboom 
ah, too bad, wish i knew about that before coding my app :-D
I will find another way no problem.

If i had the time, and the skill to get the RTC running, i certainly would give it a shot.
But unfortunately i am not that lucky, that's why you have a nice profile picture, and i have this standard icon ;)
